GetandTake/Core/Helper/FileHelper.cs

```diff
@@ -0,0 +1,65 @@
+﻿namespace GetandTake.Core.Helper;
+
+public class FileHelper
+{
+    public static string Add(IFormFile file)
+    {
+        var result = NewPath(file);
+        try
+        {
+            var sourcePath = Path.GetTempFileName();
+            if (file.Length > 0)
+            {
+                using (var stream = new FileStream(sourcePath, FileMode.Create))
+                {
+                    file.CopyTo(stream);
+                    File.Move(sourcePath, result.NewFilePath);
+                }
+            }
+        }
+        catch (Exception exception)
+        {
+
+            return exception.Message;
+        }
+
+        return result.FilePath;
+    }
+
+    public static string Update(string sourcePath, IFormFile file)
+    {
+        var result = NewPath(file);
+        try
+        {
+            if (sourcePath.Length > 0)
+            {
+                using var stream = new FileStream(result.NewFilePath, FileMode.Create);
+                file.CopyTo(stream);
+            }
+            File.Delete(sourcePath);
+        }
+        catch (Exception exception)
+        {
+
+            return exception.Message;
+        }
+
+        return result.FilePath;
+    }
+
+    public static void Delete(string path)
+    {
+        File.Delete(path);
+    }
+
+    public static (string NewFilePath, string FilePath) NewPath(IFormFile file)
```

Comment:
1. According to best practices case, methods names **should** always contain verb in their names to specify their aim.
2. Also, in my opinion, this method violates SOLID principle: despite new file path you are also generating some path and it's not really clear what does it mean (at least from names). As far as I understand, this is full and relative paths. I suggest you can just return `$"\\img\\{newPath}"` value (please, also give a clear name for it) and handle full path directly in `Add`/`Update` methods

Reply:
editted all filehelper